app/controllers/api/v1/conversations/messages_controller.rb

```diff
@@ -0,0 +1,40 @@
+module Api
+  module V1
+    module Conversations
+      class MessagesController < Api::V1::ApiController
+        helper_method :conversation, :user
+
+        def create
+          @message = conversation.messages
+                                 .create!(message_params)
+          @message.update!(user_id: user.id)
```

Comment:
Instead of acceding two times to the database (first for creating and then for updating the user_id), why can't you make something like:
```ruby
conversation.messages
                      .create!(message_params.merge({ user_id: user.id }))
```
Also I'm not sure if this update is needed, are you sure it is? Since you are already getting this conversation from the current_user.

Reply:
That `merge` refactor is done on PR #17, ill rewrite it here as well.
Also, for some reason I have no choice but to make the merge, otherwise it doesn't register the `user_id`, even when conversation is from `current_user` @gonzaM @leandrodzp